schema/training-pil.js

```diff
@@ -25,14 +25,28 @@ class TrainingPil extends BaseModel {
     };
   }
 
+  static list({ trainingCourseId, sort = {}, limit, offset }) {
+    let query = this.query()
+      .where({ trainingCourseId })
+      .withGraphFetched('profile');
+
+    if (sort.column) {
+      query = this.orderBy({ query, sort });
+    }
+
+    query = this.paginate({ query, limit, offset });
+
+    return query;
+  }
```

Comment:
Is this meant to be here?

Reply:
probably not, I forgot I wasn't on master. I can pull this out